config/initializers/content_store.rb

```diff
@@ -0,0 +1,14 @@
+require 'gds_api/content_store'
+
+class GdsApi::ContentStore::Fake
+  def content_item!(tag)
+    Struct.new("ContentItem", :content_id)
+    Struct::ContentItem.new("#{tag}-content-id")
+  end
+end
+
+Whitehall.content_store = if Rails.env.test?
+                            GdsApi::ContentStore::Fake.new
+                          else
+                            GdsApi::ContentStore.new(Plek.find('content-store'))
```

Comment:
I'm hoping we don't have to use the content-store here at all: https://github.com/alphagov/publishing-api/pull/230